jax/_src/lax/control_flow/loops.py

```diff
@@ -22,7 +22,7 @@
 import weakref
 from jax._src import core
 from jax._src import linear_util as lu
-from jax import config
+from jax import config  # type: ignore[no-redef]
```

Comment:
Not sure why, but something in this CL led to this new mypy false positive. This comment makes the typechecks pass.